package.json

```diff
@@ -43,38 +43,38 @@
     "semantic-release": "semantic-release"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.11.5",
-    "@commitlint/cli": "^9.1.2",
-    "@commitlint/config-conventional": "^9.1.2",
-    "@rollup/plugin-alias": "^3.1.1",
-    "@rollup/plugin-babel": "^5.2.0",
+    "@babel/preset-env": "^7.13.10",
+    "@commitlint/cli": "^12.0.1",
+    "@commitlint/config-conventional": "^12.0.1",
+    "@rollup/plugin-alias": "^3.1.2",
+    "@rollup/plugin-babel": "^5.3.0",
     "@rollup/plugin-buble": "^0.21.3",
-    "@rollup/plugin-commonjs": "^15.0.0",
+    "@rollup/plugin-commonjs": "^17.1.0",
     "@rollup/plugin-inject": "^4.0.2",
-    "@rollup/plugin-node-resolve": "^9.0.0",
-    "@rollup/plugin-replace": "^2.3.3",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "@rollup/plugin-replace": "^2.4.1",
     "babel-eslint": "^10.1.0",
     "babel-plugin-transform-polyfills": "^2.3.0",
-    "codecov": "^3.7.2",
-    "commitizen": "^4.2.1",
-    "core-js": "^3.6.5",
-    "cross-env": "^7.0.2",
+    "codecov": "^3.8.1",
+    "commitizen": "^4.2.3",
+    "core-js": "^3.9.1",
+    "cross-env": "^7.0.3",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^7.8.1",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint": "^7.21.0",
+    "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-better": "^0.1.5",
     "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-require-in-package": "^1.0.3",
-    "husky": "^4.3.0",
-    "jest": "^26.4.2",
-    "livereload": "^0.9.1",
+    "husky": "^5.1.3",
```

Comment:
`husky@5` is in preview mode for OSS and required additional setup compared to `v4`, so this change without config change will break the precommit hooks. I'd stay at 4 for now